charts/sda-svc/test/trigger-ingest.py

```diff
@@ -47,13 +47,18 @@
 
 if mq_connection.startswith('amqps'):
```

Comment:
i think we are doing a double check here `if tls == 'true'` and lower we check if mq.connection starts with `amqps` which is basically the same thing, maybe we can simplify that (not sure we will ever have `tls = true` and `amqp` protocol.

Reply:
I touched as little as possible in the test suite.
As for the python stuff, it needs to be rewritten so we can test everything properly, even in a complete standalone installation with the orchestrator.

Reply:
fair enough, then we can have that as part of another PR